source/Calamari.Shared/Integration/Packages/NuGet/NupkgExtractor.cs

```diff
@@ -41,25 +41,17 @@ public int Extract(string packageFile, string directory, bool suppressNestedScri
                         continue;
 
                     var targetFile = Path.Combine(targetDirectory, Path.GetFileName(unescapedKey));
-                    entry.WriteToFile(targetFile, new ExtractionOptions { Overwrite = true, WriteSymbolicLink = WriteSymbolicLink });
+                    entry.WriteToFile(targetFile, new PackageExtractionOptions { ExtractFullPath = false });
```

Comment:
I couldn't figure out what `ExtractFullPath` does, so set it to what it was before. The `PreserveFileTime` is set to `true` with this change which seems safe.

Reply:
I was wrong https://build.octopushq.com/viewLog.html?buildId=1284990&buildTypeId=OctopusDeploy_CalamariNetCore_MonoTesting_TestAmazonLinux2Mono442